keadm/app/cmd/util/edgecoreinstaller.go

```diff
@@ -18,19 +18,30 @@ package util
 
 import (
 	"bytes"
+	"encoding/json"
 	"fmt"
+	"io"
 	"io/ioutil"
+	"net/http"
 	"os"
-	"os/exec"
 	"strings"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/google/uuid"
```

Comment:
nit: lets move this import with the vendor imports

Reply:
Done. Thanks